src/nvim/if_cscope.c

```diff
@@ -1062,7 +1062,7 @@ static int cs_find_common(char *opt, char *pat, int forceit, int verbose,
       if (qf_init(wp, tmp, (char_u *)"%f%*\\t%l%*\\t%m",
               *qfpos == '-', cmdline) > 0) {
         if (postponed_split != 0) {
-          win_split(postponed_split > 0 ? postponed_split : 0,
+          (void)win_split(postponed_split > 0 ? postponed_split : 0,
```

Comment:
I'm not clear on what should happen if the split fails here. I don't think it would really be a problem, but maybe one should return an error and notify the user.